packages/itwin/tree-widget/src/components/trees/index.ts

```diff
@@ -4,13 +4,17 @@
 *--------------------------------------------------------------------------------------------*/
 
 export * from "./Common";
-export * from "./imodel-content-tree/IModelContentTree";
-export * from "./imodel-content-tree/IModelContentTreeComponent";
 export * from "./VisibilityTreeEventHandler";
 export * from "./VisibilityTreeRenderer";
+
+export * from "./imodel-content-tree/IModelContentTree";
+export * from "./imodel-content-tree/IModelContentTreeComponent";
+
 export * from "./category-tree/CategoriesTree";
 export * from "./category-tree/CategoriesTreeComponent";
 export * from "./category-tree/CategoryVisibilityHandler";
+
 export * from "./models-tree/ModelsTree";
 export * from "./models-tree/ModelsTreeComponent";
 export * from "./models-tree/ModelsVisibilityHandler";
+export * from "./models-tree/Utils";
```

Comment:
I think this export can be removed. Everything in Utils.ts is internal so there is no need to reexport it through barrel file.

Reply:
Our perf tests are going to use this internal API to get the ruleset. We could access the module directly, but doing that through the barrel is cleaner and has smaller chance of breaking our perf tests in case the API is moved to another file.